test/cypress/integration/settings/switch-psp-onboarding-cy.test.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const userStubs = require('../../stubs/user-stubs')
+const gatewayAccountStubs = require('../../stubs/gateway-account-stubs')
+const stripeAccountSetupStubs = require('../../stubs/stripe-account-setup-stub')
+
+const userExternalId = 'cd0fa54cf3b7408a80ae2f1b93e7c16e'
+const gatewayAccountId = '42'
+const gatewayAccountExternalId = 'a-valid-external-id'
+
+
+function getUserAndAccountStubs (paymentProvider , providerSwitchEnabled, gatewayAccountCredentials, merchantDetails,) {
+  return [
+    userStubs.getUserSuccess({ gatewayAccountId, userExternalId, merchantDetails }),
+    gatewayAccountStubs.getGatewayAccountByExternalIdSuccess({ gatewayAccountId, gatewayAccountExternalId, providerSwitchEnabled, paymentProvider, ...gatewayAccountCredentials && { gatewayAccountCredentials } })
+  ]
+}
+
+describe('Switch PSP settings page', () => {
+  beforeEach(() => {  
+    Cypress.Cookies.preserveOnce('session', 'gateway_account')
+  })
+
+  describe('When enable ENABLE_STRIPE_ONBOARDING_TASK_LIST is set to false', () => {  
+    beforeEach(() => {
+      process.env.ENABLE_STRIPE_ONBOARDING_TASK_LIST = false
+      cy.task('setupStubs', getUserAndAccountStubs('smartpay', null))
+    })
+
+    it('should not show stripe onboarding tasklist', () => {
+      cy.setEncryptedCookies(userExternalId)
+      cy.visit(`/account/${gatewayAccountExternalId}/settings`)
+      cy.get('.service-info--tag').should('not.contain', 'switching psp')
+      cy.get('#navigation-menu-switch-psp').should('have.length', 0)
+    })
+  })
+
+
+  describe('When stripe onboarding is switched to ', () => {
+    describe('Switching page', () => {
+      beforeEach(() => {
+        process.env.ENABLE_STRIPE_ONBOARDING_TASK_LIST = true
```

Comment:
For cypress tests, we can't test env variable within tests. Environment variable should be added in https://github.com/alphagov/pay-selfservice/blob/master/test/cypress/test.env

Also can't test for both false/true. So for cypress, we can enable the flag and check it.